tree/ntuple/v7/test/ntuple_metrics.cxx

```diff
@@ -89,3 +89,25 @@ TEST(Metrics, Timer)
    }
    EXPECT_GT(ctrWallTime.GetValue(), 0U);
 }
+
+TEST(Metrics, RNTupleWriter)
+{
+   std::string rootFileName{"test_ntuple_writer_metrics.root"};
+   FileRaii fileGuard(rootFileName);
+
+   auto model = RNTupleModel::Create();
+   auto int_field = model->MakeField<int>("ints");
+   auto float_field = model->MakeField<float>("floats");
+   auto ntuple = RNTupleWriter::Recreate(std::move(model), "ntuple", rootFileName);
+   EXPECT_FALSE(ntuple->GetMetrics().IsEnabled());
+   ntuple->EnableMetrics();
+   EXPECT_TRUE(ntuple->GetMetrics().IsEnabled());
+   *int_field = 0;
+   *float_field = 10.0;
+   ntuple->Fill();
+   ntuple->CommitCluster();
+   auto* page_counter = ntuple->GetMetrics().GetCounter("RNTupleWriter.RPageSinkFile.nPageCommitted");
+   ASSERT_FALSE(page_counter == nullptr);
+   // one page for the int field, one for the float field
+   EXPECT_EQ(page_counter->GetValueAsInt(), 2);
```

Comment:
Strangely enough, GTest wants to have the arguments reversed for proper error output, i.e.

```suggestion
   EXPECT_EQ(2, page_counter->GetValueAsInt());
```